src/os/mainctrl.js

```diff
@@ -1,5 +1,6 @@
 goog.provide('os.MainCtrl');
 
+goog.require('coreui.mixin.ICSanitizeMixin');
```

Comment:
Where is this coming from?

Reply:
bits-internal

Reply:
So that probably isn't going to work in OpenSphere, unless you can move it over?

Reply:
Ah, does OpenSphere need a copy or will it work in bits-internal if moved to OpenSphere?

Reply:
Ok, I moved the mixin into OpenSphere from bits and updated all the requires.

Reply:
Looks like there was some confusion, this will work from bits-internal because it's a mixin not an export. I reverted moving the mixin.